go/vt/schema/name_test.go

```diff
@@ -165,10 +166,61 @@ func TestAnalyzeInternalTableName(t *testing.T) {
 			assert.Equal(t, ts.isInternal, isInternal)
 			if ts.isInternal {
 				assert.NoError(t, err)
-				assert.True(t, IsGCUUID(uuid))
+				assert.True(t, isCondensedUUID(uuid))
 				assert.Equal(t, ts.hint, hint)
 				assert.Equal(t, ts.t, tm)
 			}
 		})
 	}
 }
+
+func TestToReadableTimestamp(t *testing.T) {
+	ti, err := time.Parse(time.UnixDate, "Wed Feb 25 11:06:39 PST 2015")
+	assert.NoError(t, err)
+
+	readableTimestamp := ToReadableTimestamp(ti)
+	assert.Equal(t, readableTimestamp, "20150225110639")
```

Comment:
The point was to validate that a transformation of `ti` lands at an expected value. So that constant should remain IMO. I did, however, switch between `readableTimestamp` and  `"20150225110639"` because the constant is the expected one.